### ISSUE TITLE
[FEATURE] Flat screen mode

### ISSUE THREAD

**Issue by Beethoven-n** [ferriarnus/VivecraftForge#242]
A flatscreen mode similar to [CitraVR](https://www.youtube.com/watch?v=Rdy9gGDFCIQ) might be better than the current Seated mode for keyboard and mouse players. You'd still get the 3d effect, but look at a screen instead, reducing the motion sickness you'd get from trying to play with KB/M and a headset.
A similar example of this was what [Minecraft: Gear VR Edition did.](https://www.youtube.com/watch?v=_rYfkDmSRKs)
No background would actually be needed for people to be able to play it, obviously, but it did go a long way towards the polish you got from GearVR Edition.

**Comment by fayer3** [comment]
yeah, I lready thought about a theater mode a while ago, not a priority though.
